src/test/scala/db/UserDBUnitTest.scala

```diff
@@ -0,0 +1,73 @@
+package db
+
+import Dao.DAO
+import models.UserType.{Admin, Customer}
+import models.Users
+import org.mockito.Mockito.when
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import java.util.UUID
+import scala.collection.mutable.ListBuffer
+
+class UserDBUnitTest extends AnyFlatSpec {
+
+  val userDB: DAO = mock[DAO]
+
+  it should "Add user should return new user added." in {
```

Comment:
`it` is always used from the second test case. First test case always describes completely what is the intent.